webviews/code-sidecar/cmux-code.css

```
@@ -0,0 +1,167 @@
+  --success-foreground: #087532;
+  --sidebar: rgb(248 248 248 / 88%);
+  --sidebar-foreground: #262626;
+  --sidebar-muted-foreground: #737373;
+  --sidebar-control-surface: rgb(255 255 255 / 72%);
+  --sidebar-row-hover: rgb(82 111 255 / 7%);
+  --sidebar-row-active: rgb(82 111 255 / 10%);
+  --sidebar-row-selected: rgb(82 111 255 / 12%);
+  --sidebar-border: #e5e5e5;
+  --sidebar-stage-fade: transparent;
+  --surface-raised: #ffffff;
+  font-family: -apple-system, BlinkMacSystemFont, "SF Pro Text", "Helvetica Neue", sans-serif;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)